lib/auth/auth_with_roles.go

```diff
@@ -3581,6 +3582,30 @@ func (a *ServerWithRoles) filterKubeServices(server types.Server) error {
 	// connect to them.
 	mfaParams := services.AccessMFAParams{Verified: true}
 
+	// Filter out agents that don't have support for moderated sessions access
+	// checking if the user has any roles that require it.
+	if hasLocalUserRole(a.context.Checker) {
+		roles := a.context.Checker.(LocalUserRoleSet)
+		minSupportedModeratedSessions := semver.New(utils.VersionBeforeAlpha("9.0.0"))
```

Comment:
Should we make this a package-level variable?

Reply:
I did previously remove this but I'm not sure why GitHub didn't update the diffview for you. It's gone now.